### PR TITLE
feat: walletLink installed for Coinbase wallet on desktop

### PR DESCRIPTION
**Pull request by Tulun**
Screen Shot 2022-02-22 at 11 45 20 AM

Signed-off-by: Tulun <jaykiraly@gmail.com>

### CODE REVIEW

**Review by daywiss — COMMENTED** [review]
lgtm but i dont have a way to test this. make sure this goes through QA @Tommy1231232

File: src/utils/constants.ts [hunk 1]
```diff
@@ -628,6 +628,17 @@ export function onboardBaseConfig(): Initialization {
           preferred: true,
         },
         { walletName: "coinbase", preferred: true },
+        {
+          walletName: "walletLink",
+          rpc: {
+            1: `https://mainnet.infura.io/v3/${process.env.REACT_APP_PUBLIC_INFURA_ID}`,
+            10: `https://optimism-mainnet.infura.io/v3/${process.env.REACT_APP_PUBLIC_INFURA_ID}`,
+            288: `https://mainnet.boba.network/`,
+            42161: `https://arbitrum-mainnet.infura.io/v3/${process.env.REACT_APP_PUBLIC_INFURA_ID}`,
```
Comment (daywiss) [inline src/utils/constants.ts:637]: dont we have constants or a utility to make these?

Comment (Tulun) [reply src/utils/constants.ts:637]: Not currently.

**Review by amateima — APPROVED** [review]

File: src/utils/constants.ts [hunk 2]
```diff
@@ -608,6 +608,13 @@ export const DEFAULT_TO_CHAIN_ID = ChainId.MAINNET;
 
 /* Onboard config */
 
+const rpc = {
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/utils/constants.ts]: Sounds good fixed.

**Review by Gamaranto — APPROVED** [review]

**Comment by daywiss** [comment]
this pr is on hold until we can figure out whats going on with coinbase wallet

**Comment by Tulun** [comment]
Going to close PR until we figure out the issue.

**Comment by daywiss** [comment]
this is no longer a problem, but there are other multichain related issues with coinbase wallet that we are going to wait to release this feature.

**Pull request closed without merging**